update-tags.mjs

```diff
@@ -0,0 +1,47 @@
+import { readFile, writeFile } from 'fs/promises'
+import { exec } from 'child_process'
+import { glob as baseGlob } from 'glob'
+import packageJSON from './package.json' assert { type: 'json' }
```

Comment:
Is the following legal?

```suggestion
import { version } from './package.json' assert { type: 'json' }
```

Reply:
This threw errors when I tried to implement it, something along the lines of package.json not having a valid export named version

Reply:
What errors?

Reply:
```
import { version } from './package.json' assert { type: 'json' }
         ^^^^^^^
SyntaxError: The requested module './package.json' does not provide an export named 'version'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:189:5)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
    at async loadESM (node:internal/process/esm_loader:91:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12)
```

Reply:
Oh, but of course! There is no `version` key in your `package.json` file. So add one!

Reply:
Unfortunately that error happens with a `version` key as well:
![image](https://github.com/nasa-gcn/gcn-schema/assets/11023698/7ec4a9ba-f5d7-47fe-a88a-5f6306795b86)

Reply:
How interesting! Well, thanks anyway for trying.